<launch>
  <param name="/drivers/smi540/rate_Hz" value="20.0" type="double" />
  <node name="smi540_cluster" pkg="smi540" type="smi540" output="screen" />
</launch>
